scripts/download_and_symlink.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TESTCASE_DIRECTORY="testcases/"
+
+cd "$(dirname "$(realpath $0)")/.."
+
+mkdir -p .objects
+
+echo Fetching latest object state...
+gsutil -m rsync gs://jxl-conformance/objects/ .objects/
+
+for t in $(find $TESTCASE_DIRECTORY -name test.json)
+do
+  ./scripts/list_shas.py $t | while read name sha
+  do
+    f=$(dirname $t)/$name
+    rm -f $f
+    echo Linking $name
+    ln -s $(realpath --relative-to $(dirname $f) .objects/$sha) $f
```

Comment:
to avoid the symlink issue, could we have the conformance runner check for the file in the local path, and if not there go look for it in the `.objects` directory?

Reply:
Which issue are you talking about? I guess we could do this, but I am not sure if we need to.

Reply:
running this on windows for example in the GH workflow here.

Reply:
I *believe* ln should work on Windows assuming it is installed (and it likely is if bash is).
AFAIU git doesn't support symlinks properly on Windows, but they exist.